packages/quick_actions/quick_actions_ios/example/ios/RunnerTests/Mocks/MockBinaryMessenger.swift

```diff
@@ -0,0 +1,40 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import Flutter
+
+@testable import quick_actions_ios
+
+class MockBinaryMessenger: NSObject, FlutterBinaryMessenger {
```

Comment:
The need for this is exposing a gap in the Swift generator; could you leave TODO to remove this when https://github.com/flutter/flutter/issues/136811 is implemented?